packages/aws-cdk-migration/lib/rewrite.ts

```diff
@@ -8,6 +8,11 @@ export interface RewriteOptions {
    * Optional module names that should result in replacing to something different than just 'aws-cdk-lib'.
    */
   readonly customModules?: { [moduleName: string]: string };
+
+  /**
+   * Optional flag to set for rewriting imports in alpha packages. When true, this will rewrite imports of generated L1s to reference aws-cdk-lib.
+   */
+  readonly isInAlphaPackage?: boolean;
```

Comment:
Name feels leaky. Not sure `aws-cdk-migration` should be aware of alpha packages.
A more direct name for what this feature does would be better.

Something like `useCfnConstructs` or `useBaseCfnConstructs`?

Reply:
I opted for `rewriteCfnImports` to keep it as specific as possible. Let me know what you think